oauth2client/client.py

```diff
@@ -110,8 +110,8 @@
     GCE_METADATA_TIMEOUT = 3
 
 _SERVER_SOFTWARE = 'SERVER_SOFTWARE'
-_GCE_METADATA_HOST = '169.254.169.254'
-_METADATA_FLAVOR_HEADER = 'Metadata-Flavor'
+_GCE_METADATA_URI = 'http://169.254.169.254'
+_METADATA_FLAVOR_HEADER = 'metadata-flavor'  # lowercase header
```

Comment:
> the significance of `# lowercase header`

The previous value broke `httplib2` (headers in `httplib2` are all lowercased so avoid allowing case-insensitive header lookups)

> How essential is "_HEADER" in this name?

Making the name shorter just to be short is a loss for code readability. The name describes what the variable is.